### PR TITLE
test: Unskip a flaky feed API test

### PR DESCRIPTION
**Pull request by ukstv**
Ditto.

### CODE REVIEW

File: packages/core/src/__tests__/ceramic-feed.test.ts [hunk 1]
```diff
@@ -114,14 +113,14 @@ describe('Ceramic feed', () => {
       return emissions.length >= 4 // Recon gives you more events :(
     }, 5000)
     // model.create
-    expect(emissions[0].commitId).toEqual(model.allCommitIds[0])
+    expect(String(emissions[0].commitId)).toEqual(String(model.allCommitIds[0]))
```
Comment (stbrody) [inline packages/core/src/__tests__/ceramic-feed.test.ts:116]: hmm, can you explain what was going wrong here and why casting to a string prevents the stack overflow?

Comment (ukstv) [reply packages/core/src/__tests__/ceramic-feed.test.ts:116]: That is an experimental (let me be honest here) approach based on some previous experience. Loong time ago we saw something similar with Jest unable in some cases to correctly compare StreamIDs or CommitIDs into.

Comment (stbrody) [reply packages/core/src/__tests__/ceramic-feed.test.ts:116]: Okay.  Not ideal, but worth giving it a shot and seeing how it fairs

**Review by stbrody — APPROVED** [review]

**Review by JulissaDantes — APPROVED** [review]
Thank you!

**Pull request merged**